content/en/agent/autodiscovery/_index.md

```diff
@@ -22,69 +22,51 @@ further_reading:
 
-  * [Using auto-configuration file shipped with the Agent][5]
-  * [Using a configuration file mounted within the Agent][6]
-  * [Using Key-Value Store][7]
-  * [Using Kubernetes Annotations][8]
-  * [Using Docker Labels][9]
+1. [Enable Autodiscovery](#enable-autodiscovery) for your Datadog Agent.
+2. Create [integration-specific configuration templates](#integration-templates) for each service you wish to monitor. Note that Datadog provides auto-configuration templates for [some common containerized services][3], including Apache and Redis.
 
-2. **Apply an integration template to a specific container**: Unlike in a traditional Agent setup, the Agent doesn't run all checks all the time; it decides which checks to enable by inspecting all containers running on the same host as the Agent and the corresponding loaded integration templates. The Agent then watches for Kubernetes/Docker events&mdash;container creation, destruction, starts, and stops&mdash;and enables, disables, and regenerates static check configurations on such events. As the Agent inspects each running container, it checks if the container matches any of the [Autodiscovery container identifiers][3] from any loaded integration templates. For each match, the Agent generates a static check configuration by substituting the [Template Variables][10] with the matching container's specific values. Then it enables the check using the static configuration.
+### Enable Autodiscovery 
 
-## How to set it up
-
-If running the Agent as a binary on a host, enable Autodiscovery with the [Agent](?tab=agent) tab instructions. If running the Agent as a container, enable Autodiscovery with the [Containerized Agent](?tab=containerizedagent) tab instructions.
-
-### Docker Autodiscovery
+#### With the Agent on a host
 
 {{< tabs >}}
-{{% tab "Agent" %}}
+{{% tab "Docker" %}}
 
-To enable Autodiscovery over Docker containers, add the following configuration block in the [Agent `datadog.yaml` configuration file][1]:
+Add the following configuration block in the `datadog.yaml` [configuration file][1]. 
 
-```yaml
+```
 listeners:
   - name: docker
 config_providers:
   - name: docker
     polling: true
 ```
 
-[1]: /agent/guide/agent-configuration-files/?tab=agentv6#agent-main-configuration-file
-{{% /tab %}}
-{{% tab "Containerized Agent" %}}
-
-To automatically enable Autodiscovery over Docker containers, mount `/var/run/docker.sock` into the Containerized Agent.
 
+[1]: /agent/guide/agent-configuration-files/?tab=agentv6#agent-main-configuration-file
 {{% /tab %}}
-{{< /tabs >}}
+{{% tab "Kubernetes" %}}
 
-### Kubernetes Autodiscovery
+Add the following configuration block in the `datadog.yaml` [configuration file][1]. 
 
-{{< tabs >}}
-{{% tab "Agent" %}}
-
-To enable Autodiscovery over containers within Kubernetes, add the following configuration block in the [Agent `datadog.yaml` configuration file][1]:
-
-```yaml
+```
 listeners:
```

Comment:
I don't think that you need explicit configuration to enable autodiscovery. AFAIK it is enabled by default when deployed on Kubernentes, see for example the blog post that shows how to monitor EKS that doesn't mention these options: https://www.datadoghq.com/blog/eks-monitoring-datadog/

Reply:
added a note that AD is enabled by default on K8s